python/foglamp/plugins/south/coap/coap.py

```diff
@@ -95,6 +96,7 @@ def plugin_init(config):
         handle: JSON object to be used in future calls to the plugin
     Raises:
     """
+    _LOGGER.debug("plugin_init called")
```

Comment:
I was in favor to remove such comments and test patched calls. Instead of adding new

Reply:
Such debug logs help in code coverage whereas method patching bypasses method body. Hence debug logs are preferred. Patching should be used only for those methods which play a secondary role in the method under testing, e.g. logger.